packages/yoga/src/Accordion/web/Accordion.jsx

```diff
@@ -1,19 +1,33 @@
 import React, { useState } from 'react';
 import styled, { css } from 'styled-components';
 import { string, node, bool } from 'prop-types';
+import { lineHeights } from '@gympass/yoga-tokens/src/global';
```

Comment:
Will this import work after building? Because I think that the `src` folder won't be available after building the `@gympass/yoga-tokens`.

Reply:
Good point. I noticed that using line Height.medium is unnecessary to set the lineHeight in the Text component.
Fixed: https://github.com/gympass/yoga/pull/628/commits/87a7c80323654b91de836583300ad2d4a69ead13